addons/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/config/PLCLogoDigitalConfiguration.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.plclogo.config;
+
+/**
+ * The {@link PLCLogoDigitalConfiguration} holds configuration of Siemens LOGO! PLC
+ * digital input/outputs blocks.
+ *
+ * @author Alexander Falkenstern - Initial contribution
+ */
+public class PLCLogoDigitalConfiguration extends PLCLogoBlockConfiguration {
+
+    public PLCLogoDigitalConfiguration() {
+        super();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean isBlockValid() {
+        boolean valid = false;
```

Comment:
If you mean familiar to PLCLogoAnalogConfiguration::isBlockValid, then no. Block names decides about types to handle. Please compare `valid = valid || name.startsWith("I") || name.startsWith("NI"); // Inputs` from PLCLogoDigitalConfiguration and `valid = valid || name.startsWith("AI") || name.startsWith("NAI"); // Inputs` from PLCLogoAnalogConfiguration. The first case handles bits, second handles words (i.e 2 bytes)